roles/container-engine/containerd/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: containerd | Install required packages
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - libseccomp2
+    - ethtool
+
+- name: containerd | Download release tarball
+  get_url:
```

Comment:
Download should be moved to download role i suppose

Reply:
+1 on that

Reply:
That would be a bit problematic since the download role is run after the container-engine role and the downloading of containers relies pretty much on a container-engine

Reply:
Moved downloads to download role